vendor/github.com/hashicorp/vault/api/auth_token.go

```diff
@@ -272,4 +272,5 @@ type TokenCreateRequest struct {
 	NumUses         int               `json:"num_uses"`
 	Renewable       *bool             `json:"renewable,omitempty"`
 	Type            string            `json:"type"`
+	EntityAlias     string            `json:"entity_alias"`
```

Comment:
Should this be updated here or through some sort of update using go mod?

Reply:
This was generated/updated by go mod. I removed it now since vendor update happens now before release. So no need to do that manually here.